### PR TITLE
py-sacrebleu

### PR DESCRIPTION
**Pull request by Sangu-Mbekelu**

### CODE REVIEW

**Comment by Sangu-Mbekelu** [comment]
@spackbot fix style

File: var/spack/repos/builtin/packages/py-sacrebleu/package.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-sacrebleu/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-sacrebleu/package.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    depends_on("python@3.6.0:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-portalocker", type=("build", "run"))
+    depends_on("py-regex", type=("build", "run"))
+    depends_on("py-tabulate@0.8.9:", type=("build", "run"))
+    depends_on("py-numpy@1.17:", type=("build", "run"))
+    depends_on("py-colorama", type=("build", "run"))
+
+    def patch(self):
+        touch = which("touch")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-sacrebleu/package.py]: This line isn't needed.  Every package automatically gets access to Spack's `touch()` along with other commands from https://spack.readthedocs.io/en/latest/llnl.util.html#module-llnl.util.filesystem that do a little extra work.

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage, branch, and version.  Just a couple of minor points, especially the copyright year).

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Please fix the ending copyright year.

**Review by adamjstewart — DISMISSED** [review]
LGTM once @tldahlgren's changes are made

**Review by adamjstewart — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Comment by tldahlgren** [comment]
FWIW. I'm getting an odd error that doesn't appear to be related to the package AFAICT when  I try to install it:

```
$ spack install py-sacrebleu
==> Error: VersionBase('7')
```

**Pull request merged**